packages/fluentui/docs/src/prototypes/Calls/participant.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import { Avatar, Flex, Text, Animation, MenuButton, FlexItem } from '@fluentui/react-northstar';
+import { MoreIcon, MicOffIcon, ParticipantRemoveIcon, MicIcon } from '@fluentui/react-icons-northstar';
+
+const Participant: React.FC<{
+  image?: string;
+  name: string;
+  status: object;
+  role: string;
+  isTalking?: boolean;
+  isMuted?: boolean;
+}> = ({ image, name, status, role, isTalking, isMuted }) => {
+  const [muted, setMuted] = React.useState(isMuted);
+  const [talking, setTalking] = React.useState(isTalking);
+  const [openMenu, setOpenMenu] = React.useState(false);
+
+  return (
+    <Flex
+      fill
+      gap="gap.small"
+      space="between"
+      onContextMenu={e => {
+        setOpenMenu(!openMenu);
+        e.preventDefault();
+      }}
+      variables={{ isRelative: true }}
+    >
+      {talking ? (
+        <Animation name="talking">
+          <Avatar
+            variables={{ isTalking: true }}
+            image={`data:image/jpeg;base64,${image}`}
+            name={name}
+            status={status}
+          />
+        </Animation>
+      ) : (
+        <Avatar image={`data:image/jpeg;base64,${image}`} name={name} status={status} />
+      )}
+
+      <Flex column>
+        <Text weight={talking && 'bold'} content={name} variables={{ isNameText: true }} />
+        <Text variables={{ isRoleText: true }} content={role} />
+      </Flex>
+
+      {muted ? (
+        <>
+          <FlexItem push>
+            <MicOffIcon />
+          </FlexItem>
+          <MenuButton
```

Comment:
Seems like the `MenuButton` is rendered anyway. I would suggest rendering it after the condition. Something like:

{muted && <FlexItem>...</FlexItem>}
<MenuButton ... />